adafruit_ducky.py

```diff
@@ -128,76 +127,76 @@ def __init__(
         self.layout = layout
         self.lines = []
         self.default_delay = 0
-        self.last = 0
+        self.prev_line = None
+        self.next_index = 0
+        self.wait_until = 0
+        self.repeat = 0
 
         with open(filename, "r") as duckyscript:
             for line in duckyscript:
-                self.lines.append(line[:-1].rstrip("\r"))
+                line = line.strip(" \r")
```

Comment:
Is the space before the `\r` intentional? Is that perhaps meant to be a newline like this:
```
line = line.strip("\n\r")
```

Reply:
The space is there to remove spaces from the beginning and end of the line to avoid empty commands from the later split on space on line 148.  But I guess there should be a \n there too.

Reply:
Added \n and pushed it...